src/lib/init.ts

```diff
@@ -539,143 +562,142 @@ function stringifyError(e${isJS ? "" : ": unknown"}) {
-  timeWindow: "1 minute",
-});
+const PEER_PRIVATE_KEY_BYTES = new TextEncoder().encode(PEER_PRIVATE_KEY);
 
-server.addHook("onReady", async () => {
-  await Fluence.connect(relays[0], {
-    keyPair: {
-      type: "Ed25519",
-      source: PEER_PRIVATE_KEY,
-    },
+export default async function (server${isJS ? "" : ": FastifyInstance"}) {
+  await server.register(import("@fastify/rate-limit"), {
+    max: 100,
+    timeWindow: "1 minute",
   });
-});
 
-server.addHook("onRequest", async (request, reply) => {
-  if (request.headers.access_token !== ACCESS_TOKEN) {
-    await reply.status(403).send({
-      error: "Unauthorized",
-      statusCode: 403,
+  server.addHook("onReady", async () => {
+    await Fluence.connect(relays[0], {
+      keyPair: {
+        type: "Ed25519",
+        source: PEER_PRIVATE_KEY_BYTES,
+      },
     });
-  }
-});
-
-server.addHook("onClose", async () => {
-  await Fluence.disconnect();
-});
+  });
 
-const callbackBody = Type.Object({
-  name: Type.String(),
-});
+  server.addHook("onRequest", async (request, reply) => {
+    if (request.headers.access_token !== ACCESS_TOKEN) {
+      await reply.status(403).send({
+        error: "Unauthorized",
+        statusCode: 403,
+      });
+    }
+  });
 
-${isJS ? "" : "type callbackBodyType = Static<typeof callbackBody>;"}
+  server.addHook("onClose", async () => {
+    await Fluence.disconnect();
+  });
 
-const callbackResponse = Type.String();
+  const callbackBody = Type.Object({
+    name: Type.String(),
+  });
 
-${isJS ? "" : "type callbackResponseType = Static<typeof callbackResponse>;"}
+  ${isJS ? "" : "type callbackBodyType = Static<typeof callbackBody>;"}
 
-const showSubnetResponse = Type.Array(
-  Type.Object({
-    host_id: Type.Union([Type.String(), Type.Null()]),
-    services: Type.Union([Type.Array(Type.String()), Type.Null()]),
-    spells: Type.Union([Type.Array(Type.String()), Type.Null()]),
-    worker_id: Type.Union([Type.String(), Type.Null()]),
-  }),
-);
+  const callbackResponse = Type.String();
 
-${
-  isJS ? "" : "type showSubnetResponseType = Static<typeof showSubnetResponse>;"
-}
+  ${isJS ? "" : "type callbackResponseType = Static<typeof callbackResponse>;"}
 
-const runDeployedServicesResponse = Type.Array(
-  Type.Object({
-    answer: Type.Union([Type.String(), Type.Null()]),
-    worker: Type.Object({
-      host_id: Type.String(),
-      pat_id: Type.String(),
+  const showSubnetResponse = Type.Array(
+    Type.Object({
+      host_id: Type.Union([Type.String(), Type.Null()]),
+      services: Type.Union([Type.Array(Type.String()), Type.Null()]),
+      spells: Type.Union([Type.Array(Type.String()), Type.Null()]),
       worker_id: Type.Union([Type.String(), Type.Null()]),
     }),
-  }),
-);
+  );
 
-${
-  isJS
-    ? ""
-    : "type runDeployedServicesResponseType = Static<typeof runDeployedServicesResponse>;"
-}
+  ${
+    isJS
+      ? ""
+      : "type showSubnetResponseType = Static<typeof showSubnetResponse>;"
```

Comment:
I think it's more or less accepted convention to name types starting with an uppercase letter
```suggestion
      : "type ShowSubnetResponseType = Static<typeof showSubnetResponse>;"
```